src/screens/SignUp/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import i18next from 'i18next';
+
+import SignUpForm from '../../components/forms/SignUpForm';
+
+import styles from './styles.module.scss';
+
+function SignUp() {
+  return (
+    <div className={styles.container}>
+      <div className={styles.wrapper}>
+        <div className={styles.card}>
+          <div className={styles.logo} />
+          <SignUpForm />
+          <div>
+            <button className="button__redirect" type="button">
```

Comment:
@Juliocho990 can also do it this way and like this 
```suggestion
            <button className="buttonRedirect" type="button">
```

In the style file you use the class like this
```
            .button-redirect {
            }
```

that is the standard that we use in Wolox to react

Reply:
👍 Ok! I got it.

Reply:
sorry, it was my mistake, I looked at the classes wrongly and I got confused, as you have it, I look very good